pulser-core/pulser/register/weight_maps.py

```diff
@@ -63,7 +65,7 @@ def __init__(
     @property
```

Comment:
Can't there be use cases where we wan't to modify the weights of the detuning map ? Such that weights could be typing.Sequence[float | pm.Differentiable] ?

Reply:
I agree, let's keep it "simple" 😄